src/index.rs

```diff
@@ -715,6 +698,26 @@ mod tests {
         (file, file_path)
     }
 
+    fn modify_file(file: &mut File) {
+        let bytes = rand::thread_rng().gen::<[u8; MODIFIED_SIZE as usize]>();
+        file.write(&bytes)
+            .expect("Couldn't write into the file");
+    }
+
+    fn resource_id_1() -> ResourceId {
+        ResourceId {
+            data_size: DATA_SIZE_1,
+            crc32: CRC32_1,
+        }
+    }
+
+    fn resource_id_2() -> ResourceId {
+        ResourceId {
+            data_size: DATA_SIZE_2,
+            crc32: CRC32_2,
+        }
```

Comment:
Let's define these resources as constants instead

```suggestion
   const RESOURCE_ID_1: ResourceId = ResourceId {
        data_size: DATA_SIZE_1,
        crc32: CRC32_1,
    };

    const RESOURCE_ID_2: ResourceId = ResourceId {
        data_size: DATA_SIZE_2,
        crc32: CRC32_2,
    };
```